index.html

```diff
@@ -23,7 +23,7 @@ <h1 class="logo">My log</h1>
               </div>        
               <ul class="humber">
                         <li>
-                          <img class="vector" src="./images/Img/Union.png" alt="Union">
+                          <img class="vector" src="./images/Img/Union.png" alt="humbugger-icon">
                         </li>
                         <li class="menu">
                           <a href="#">Portfolio</a>
```

Comment:
- [ ] Since I can not be able to access the Logo changes, I think it would be a good option to attach the comment here, though it refers to the page logo. If you notice most of the websites have the logo such that when clicked on, the page is redirected to the Homepage. Kindly, I suggest to consider wrapping the Logo in the `a` tag and ensure that when clicked on, you are taken to the home page of the portfolio. This will help to increase the accessibility of the portfolio and therefore more user-friendly ☺️.